torch/nn/utils/rnn.pyi

```diff
@@ -33,17 +33,17 @@ class PackedSequence(PackedSequence_):
         sorted_indices: Optional[Tensor] = ...,
         unsorted_indices: Optional[Tensor] = ...,
     ) -> Self: ...
-    def pin_memory(self: _T) -> _T: ...
```

Comment:
Is `_T` even needed in this file anymore? Just remove it from the Generic parent class?

Reply:
Good catch! When I ran ruff check locally it didn't mark these functions because of the @overload